python/lsst/ctrl/bps/drivers.py

```diff
@@ -129,10 +131,17 @@ def _init_submission_driver(config_file, **kwargs):
     else:
         _LOG.debug("Skipping submission checks.")
 
-    # make submit directory to contain all outputs
-    submit_path = config["submitPath"]
-    os.makedirs(submit_path, exist_ok=True)
-    config[".bps_defined.submitPath"] = submit_path
+    # Make submit directory to contain all outputs.
+    submit_path = Path(config["submitPath"])
+    try:
+        submit_path.mkdir(parents=True, exist_ok=False)
+    except OSError as exc:
+        if exc.errno == errno.EEXIST:
+            reason = "Directory already exists"
```

Comment:
Is the reason to handle this separately because otherwise it says "File exists"?  I know we discussed being able to tell whether the error was about it existing or not, but the way you wrote line 143 is generic enough to cover both.

Reply:
Yes, I wrote it that way because I thought that "File exists" in this context may be a bit confusing for the user. Would you like me to change it? (Remove the if-else statement and just use `exc.strerror` in line 143.)

Reply:
I don't have a strong opinion either way.  Just be aware that the error type still is FileExistsError and shows up at the beginning of that error line, but your message does reinforce that it's the submit directory that already exists.

Reply:
I leave it as is then.